app/src/main/java/org/gnucash/android/db/adapter/AccountsDbAdapter.java

```diff
@@ -1176,7 +1176,7 @@ public boolean isFavoriteAccount(String accountUID){
 
                 Transaction transaction = new Transaction(GnuCashApplication.getAppContext().getString(R.string.account_name_opening_balances));
                 transaction.setNote(getAccountName(accountUID));
-                transaction.setCurrencyCode(currencyCode);
```

Comment:
The currency_code column in the transactions database might be a non-null column. Could you check?
In that case, we would need to set it to some default string or for the time-being. 
During the big db migration coming up, I'd get rid of that column.